package.json

```diff
@@ -9,58 +9,65 @@
     "prebuild": "npm run lint",
     "build": "ng build --prod",
     "pretest": "npm run lint",
-    "test": "ng test -cc -sm=false",
-    "e2e": "node e2e.ts",
+    "test": "echo 'Test suite should be run with \"`npm run tests` or npm `run tests-ci`\"'",
+    "tests": "ng test --watch=false",
```

Comment:
We should actually move to `npm run test`.  I know in EPIC we have it as `npm run tests`, but IIRC that was only because of the dockerfiles/openshift DC's running the tests on startup when they were in fact broken.  We can safely move towards 'test' after these tests pass.  NRPTI also has it as `npm run test`.